gc/gc-base/src/test/java/org/projectnessie/gc/base/AbstractRestGC.java

```diff
@@ -90,7 +90,7 @@ void fillExpectedContents(Branch branch, int numCommits, List<Row> expected)
             });
   }
 
-  void performGc(
+  protected void performGc(
```

Comment:
overriding in another module (different package name), hence it cannot be package private.

Reply:
refer here: https://github.com/projectnessie/nessie/pull/3626